DQM/HcalMonitorTasks/src/HcalDigiMonitor.cc

```diff
@@ -578,6 +578,20 @@ void HcalDigiMonitor::analyze(edm::Event const&e, edm::EventSetup const&s)
 	if (debug_>0)
 		std::cout << "### Processing FED: " << i << std::endl;
 
+	//	For uTCA spigos are useless => by default we have digisize = 4
+	//	As of 20.05.2015
+	//	HF = 2
+	if ((i>=1118 && i<=1122) ||
+			(i>=718 && i<=723))
```

Comment:
Hi David,

Regarding the 718 up to 723 - NO,  see here /afs/
cern.ch/cms/slc6_amd64_gcc491/cms/cmssw/CMSSW_7_3_6/c/DataFormats/FEDRawData/interface/FEDNumbering.h
718 up to 723 are VME FEDs for HF which need to be manually treated.

1118 and 1122 - possibly I could use the HCALuTCA boundaries (but those
span over the whole region of 200FEDs) - and you are right, they could be
used, but would be as good as what I'm putting now

VK

On Thu, May 21, 2015 at 4:09 PM, David Lange <notifications@github.com>
wrote:

> In DQM/HcalMonitorTasks/src/HcalDigiMonitor.cc
> <https://github.com/cms-sw/cmssw/pull/9199#discussion_r30805137>:
>
> > @@ -578,6 +578,20 @@ void HcalDigiMonitor::analyze(edm::Event const&e, edm::EventSetup const&s)
> >  	if (debug_>0)
> >  		std::cout << "### Processing FED: " << i << std::endl;
> >
> > +	//	For uTCA spigos are useless => by default we have digisize = 4
> > +	//	As of 20.05.2015
> > +	//	HF = 2
> > +	if ((i>=1118 && i<=1122) ||
> > +			(i>=718 && i<=723))
>
> are there no constants already defined corresponding to whatever "1118",
> "1122", "718", and "723" happen to be?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/cms-sw/cmssw/pull/9199/files#r30805137>.
>